README.md

```diff
@@ -13,38 +13,39 @@ The primary repository that this documents is [docassemble-AssemblyLine](https:/
 
 ## Contributing to the documentation
 
-This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
-Documentation is contained in a series of markdown files. The menu is contained in a file named `sidebar.js`.
+This website is built using [Docusaurus](https://docusaurus.io/), a static website generator for documentation. Documentation is contained in a series of markdown files. The menu is contained in a file named `sidebar.js`.
 
-You should have a local copy of node.js and npm installed to edit the documentation.
+If you want to contribute, please follow the [LIT Lab style guide](https://github.com/SuffolkLITLab/styleguide). When you submit a pull request with your changes, request a review from one or more [LIT Lab staff members](https://github.com/orgs/SuffolkLITLab/teams/lit-lab-staff).
 
-### Installing
+You will need to have a local copy of [node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) installed to edit the documentation.
```

Comment:
This is only if they want to see it before they submit the edit. People can just edit on GitHub, for example, to fix typos and other things that don't need complex formatting.